supervisor/create_linux.go

```diff
@@ -0,0 +1,13 @@
+package supervisor
+
+import "github.com/docker/containerd/runtime"
+
+type platformStartTask struct {
+	Checkpoint *runtime.Checkpoint
+}
+
+func setTaskCheckpoint(t *StartTask, task *startTask) {
```

Comment:
Can we have it as a `func (task *startTask) setTaskCheckpoint(t *StartTask)` instead?